AzureResourceManager/Xray/MP_submission/mainTemplate.json

```diff
@@ -267,7 +268,7 @@
     },
     {
       "type": "Microsoft.Storage/storageAccounts",
-      "apiVersion": "2019-06-01",
+      "apiVersion": "2020-08-01-preview",
```

Comment:
Please make sure we use the latest API version without preview.

Reply:
Updated apiVersion 2021-02-01